shell/platform/darwin/ios/platform_view_ios.mm

```diff
@@ -112,18 +93,16 @@ new AccessibilityBridge(static_cast<FlutterView*>(owner_controller_.get().view),
 // |PlatformView|
 std::unique_ptr<Surface> PlatformViewIOS::CreateRenderingSurface() {
   if (!ios_surface_) {
-    ios_surface_ = CreateIOSSurface();
```

Comment:
This is the problem line in this CL.  This is calling CreateIOSSurface directly from the platform thread.  The intent of this call is to be on the GPU thread.